scss/civicrm/contact/pages/_events.scss

```diff
@@ -15,6 +15,34 @@
     }
   }
 
+  #searchForm {
+    @include border-box();
+    padding: 20px;
+  }
+
+  .crm-form-submit {
```

Comment:
aren't we styling the buttons already? Why is this button not getting the style? Is the markup different? If it is, don't we have some sass mixing that we can `@include` instead of having to re-define all the values again?